bundle/Event/InformationCollected.php

```diff
@@ -67,10 +75,52 @@ public function getLocation()
      * Returns additional content
      * This can be ez content or site api content.
      *
-     * @return Content
+     * @return Content|null
      */
     public function getAdditionalContent()
     {
         return $this->additionalContent;
     }
+
+    /**
+     * @return array
+     */
+    public function getAdditionalParameters()
+    {
+        return $this->additionalParameters;
+    }
+
+    /**
+     * @param array $additionalParameters
+     */
+    public function setAdditionalParameters(array $additionalParameters)
+    {
+        $this->additionalParameters = $additionalParameters;
+    }
+
+    /**
+     * @param string $key
+     * @param mixed $value
+     * @return self
+     */
+    public function setAdditionalParameter(string $key, $value)
+    {
+        $this->additionalParameters[$key] = $value;
+
+        return $this;
+    }
+
+    /**
+     * @param string $key
+     * @return mixed
+     * @throws MissingAdditionalParameterException
+     */
+    public function getAdditionalParameterValue(string $key)
+    {
+        if (!isset($this->additionalParameters[$key])) {
```

Comment:
You should use `array_key_exists`, since `isset` will fail with `null` values.

Also, naming the method `getAdditionalParameter` is enough.

Reply:
Changed in https://github.com/netgen/NetgenInformationCollectionBundle/pull/76/commits/6f8b75e852cf8ad8de9d9dd3e24598adb510636a